packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md

```diff
@@ -9,10 +9,18 @@ description: >-
 ---
 
 
-Need to explain how to connect their app: https://dashboard.request.network/settings/apps
+Request API is an abstraction over the Request Network for easy, but centralised, access. It's the fastest way to get started and build integrations.
+
 ## Pros
-TODO
+* Simple REST API, familiar for classic web developers
+* No key management
+* Faster response time
+* Notifications on new request and received payments
+* Send requests to other API users based on their email, instead of their ETH address.
+* Easy to build applications on top of Request (see [here](./3-api-apps))
+* Easy choice between fully public data or encrypted data on the network 
```

Comment:
What do you mean ?
Do you mean it's easy to create encrypted requests ?

Reply:
yes, but the request will not be end-to-end encrypted